src/main/scala/org/broadinstitute/hail/methods/Aggregators.scala

```diff
@@ -41,10 +42,15 @@ object Aggregators {
     } else None
   }
 
-  def buildSampleAggregations(vds: VariantDataset, ec: EvalContext): Option[(String) => Unit] = {
+  def buildSampleAggregations(vds: VariantDataset, ec: EvalContext, branchingFactor: Option[Int]): Option[(String) => Unit] = {
```

Comment:
How about 50?  Laurent had issues with branching factors that were about twice that, and we've done plenty of large treeAggregates on the cray with branching factors around this.